libs/model/src/models/community_contract_template.ts

```diff
@@ -24,8 +24,16 @@ export default (
       'CommunityContractTemplate',
       {
         id: { type: dataTypes.INTEGER, autoIncrement: true, primaryKey: true },
-        community_contract_id: { type: dataTypes.INTEGER, allowNull: false },
-        cctmd_id: { type: dataTypes.INTEGER, allowNull: false },
+        community_contract_id: {
```

Comment:
All of the columns here except cctmd_id have sequences associated with them. Is that something we need to represent in the model?

Reply:
We will discuss defaults, sequences, and indexes next... probably not a big deal in the static model